src/pages/index.js

```diff
@@ -274,17 +246,43 @@ export const pageQuery = graphql`
         contentPadding
       }
     }
+    allMarkdownRemark(
+      sort: {
+        fields: [frontmatter___releaseDate, frontmatter___title]
+        order: [DESC, ASC]
+      }
+      filter: { fields: { slug: { regex: "/whats-new/" } } }
+    ) {
+      edges {
+        node {
+          id
+          frontmatter {
+            title
+            releaseDate(formatString: "MMMM D, YYYY")
```

Comment:
I don't actually think that we are using the release date on this page. We can probably remove it from the query (and the related propTypes).

```suggestion
```

Reply:
left behind when i pasted the query 😅